vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -265,87 +251,123 @@ external:
       # Default enabled to create the necessary RBAC roles
       # and role bindings in order to find the secret
       createRBAC: true
-EOF
 ```
 
-## Deploy vCluster Pro
+
+##  Install vCluster
+
+### Pre-Requisites 
+- Ability to pull from your private registry
+<BasePrerequisites />
 
 <Flow>
   <Step>
-License Setup
+  On the host cluster, create the namespace for the vCluster.
 
-Create a Kubernetes `Secret` from the License Key provided by Loft in the `Namespace` where you are installing the air-gapped vCluster instance:
+  ```bash title="Create vCluster namespace"
+  export VCLUSTER_NAMESPACE=vcluster-my-vcluster 
+  kubectl create namespace ${VCLUSTER_NAMESPACE}
+  ```
+
+  </Step>
+  <Step>
+
+  Create the secret to validate the license. 
+
+Create a Kubernetes secret from the License Key provided by LoftLabs in the namespace for the vCluster.
 
 :::tip
 License secret is already `base64` encoded.
 :::
 
-```bash title="Create Kubernetes Secret"
-#!/bin/bash
-
+```bash title="Create Kubernetes secret for License Key"
 # Set the license key as an environment variable (already base64 encoded)
 export VCLUSTER_LICENSE_KEY="YOUR_BASE64_ENCODED_LICENSE_KEY_HERE"
 
-# Optionally create the namespace
-kubectl create namespace vcluster-ns
-
 # Create secret
 kubectl create -f - <<EOF
 apiVersion: v1
 kind: Secret
 metadata:
-  name: vcluster-platform-api-key
-  namespace: vcluster-ns
+  name: vcluster-platform-license-key
+  namespace: ${VCLUSTER_NAMESPACE}
 type: Opaque
 data:
   license: ${VCLUSTER_LICENSE_KEY}
 EOF
 ```
   </Step>
-
   <Step>
-Configuration File
+**Optional** Create the secret to authenticate to the private registry.
+
+Create a Kubernetes secret with the login credentials for your private registry in the namespace for the vCluster.
+
+```bash title="Create Kubernetes secret for login credentials"
+kubectl create -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: registry-credentials-secret-name
+  namespace: ${VCLUSTER_NAMESPACE}
+type: Opaque
+data:
+  # Credentials based on your private registry
+EOF
+```
 
-At this point, you should have a `vcluster.yaml` file with the necessary
-configuration for your air-gapped vCluster Pro installation. It could look
-something like this:
+  </Step>
+
+  <Step>
+Review your `vcluster.yaml` file.
 
-Create a `vcluster.yaml` file (Helm chart values file) based on the configuration examples from above:
+Use the `vcluster.yaml` file with the necessary
+configuration for your air-gapped vCluster installation. 
 
 ```yaml title="vcluster.yaml configuration"
 controlPlane:
   advanced:
     defaultImageRegistry: my-private-registry:5000/vcluster/
+    defaultImageRegistry: ecr.io/myteam # Replace with your private image registry
```

Comment:
Its redundant here. we can remove one of `defaultImageRegistry:`

Reply:
Removed in my new draft.